modules/Core/pages/panel/registration.php

```diff
@@ -190,6 +190,10 @@
         'docLinkStart' => '<a href="https://docs.namelessmc.com/en/oauth" target="_blank">',
         'docLinkEnd' => '</a>'
     ]),
+    'REDIRECT_URL' => $language->get('admin', 'redirect_url'),
+    'CLIENT_ID' => $language->get('admin', 'client_id'),
+    'CLIENT_SECRET' => $language->get('admin', 'client_secret'),
+    'OAUTH_URL' => URL::getSelfURL() . 'oauth/&provider={{provider}}',
```

Comment:
This does not support un-friendly urls

`URL::getSelfURL() . ltrim(URL::build('/oauth/', 'provider={{provider}}'), '/'),`
^^ like we do on webhook events url

Reply:
This should be resolved now